packages/eas-cli/src/credentials/android/actions/CreateGoogleServiceAccountKeyForFcmV1.ts

```diff
@@ -0,0 +1,76 @@
+import chalk from 'chalk';
+import fs from 'fs-extra';
+
+import { AccountFragment, GoogleServiceAccountKeyFragment } from '../../../graphql/generated';
+import Log, { learnMore } from '../../../log';
+import { promptAsync } from '../../../prompts';
+import { CredentialsContext } from '../../context';
+import { GoogleServiceAccountKey } from '../credentials';
+import {
+  detectGoogleServiceAccountKeyPathAsync,
+  readAndValidateServiceAccountKey,
+} from '../utils/googleServiceAccountKey';
+
+export class CreateGoogleServiceAccountKeyForFcmV1 {
```

Comment:
This does the same thing as [CreateGoogleServiceAccountKey](https://github.com/expo/eas-cli/blob/eb9152aa9c9590ca8720496b22c6085776217428/packages/eas-cli/src/credentials/android/actions/CreateGoogleServiceAccountKey.ts#L14), so it is not necessary

Reply:
I left this as its own class because of the logging/erroring that's specific to FCM V1. I could put some conditional logic in `CreateGoogleServiceAccountKey` to render FCM-specific logs or not based on args from the caller, but this seemed cleaner to me.